src/locales/fr/egg.ts

```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "Vous n’avez pas assez de coupons !",
   "tooManyEggs": "Vous avez trop d’Œufs !",
   "pull": "Tirage",
-  "pulls": "Tirages"
+  "pulls": "Tirages",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```

Comment:
```suggestion
  "eggHatched": "Un {{pokemonName}} vient de sortir de l’Œuf !"
```

Wrong sentence in French
![pokemon-ecarlate-violet-ouisticram-tortipouss-tiplouf-illu](https://github.com/pagefaultgames/pokerogue/assets/8146474/03a7d94e-f1f8-4004-8819-8898f0b6eaa8)

Reply:
Yeah your sentence for the egg is better !